.conform.yaml

```
@@ -0,0 +1,14 @@
+policies:
+  - type: commit
+    spec:
+      conventional:
+        types:
+          - chore
```

```suggestion
          - feat
          - chore
```
Savner `feat` for nye funksjoner eller features.

---

feat og fix er innebygget i standarden, ref https://www.conventionalcommits.org/en/v1.0.0

---

La de til for det så jeg kunne kommentere hvordan de kan brukes